devIssuers.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: subpage
+title: "Developer Docs for VC Issuers"
+permalink: /developers/issuers
+---
+# CHAPI for VC Issuers
+CHAPI integrates easily into issuer websites, allowing your site to issue Verifiable Credentials and present them for storage in the recipient's digital wallet:
+
+## Resources
+- **Example Code**: the [chapi-demo-issuer](https://github.com/credential-handler/chapi-demo-issuer) contains a full example implementation and is referenced throughout this guide. 
+- **Polyfill Library**: The [credential-handler-polyfill](https://github.com/credential-handler/credential-handler-polyfill) library provides the needed functionality in the browser.
+
+## Import the CHAPI Polyfill into your Issuer Site
+If you're working in vanilla JavaScript, you can add the `navigator.credentials` and `credentialHandlerPolyfill` globals to your code and then load the polyfill library:
+
+```
+<script src="https://unpkg.com/credential-handler-polyfill@3.0.0/dist/credential-handler-polyfill.min.js"></script>
+
+<script>
+credentialHandlerPolyfill.loadOnce();
+</script>
+```
+<p class="code-annotation">
+  <a href="https://github.com/credential-handler/chapi-demo-issuer/blob/master/index.html"
+  target="_blank" rel="noopener noreferrer"> chapi-demo-issuer/index.html </a>
+</p>
```

Comment:
```suggestion
<p class="code-annotation">
  <a href="https://github.com/credential-handler/chapi-demo-issuer/blob/master/index.html"
  target="_blank" rel="noopener noreferrer"> chapi-demo-issuer/index.html </a>
</p>
```

Unsure about what this is doing?

Reply:
These are the formatted links to the relevant example code.  Concept is to show simplified code snippets inline and then point to the chapi-demo-issuer/wallet/verifier examples so readers can see a full implementation.